images/images.qrc

```diff
@@ -885,6 +885,10 @@
         <file>themes/default/mIconGoochMaterial.svg</file>
         <file>themes/default/mIconSimpleLineMaterial.svg</file>
         <file>themes/default/mIconPhongTexturedMaterial.svg</file>
+        <file>themes/default/mActionLabelAnchorCenter.svg</file>
```

Comment:
Really good job on these icons, they are looking great and easy to understand.

Reply:
Thanks! I'd like to see us add more little helper icons like this in future -- they really make it easier for new users to instantly understand settings.